receiver/jaegerreceiver/trace_receiver.go

```diff
@@ -228,6 +230,7 @@ type agentHandler struct {
 	id           config.ComponentID
 	transport    string
 	nextConsumer consumer.Traces
+	receiver     *obsreport.Receiver
```

Comment:
`obsrecv` is a better name for the variable, and everywhere else

Reply:
Changed the name in all spots in latest commit